integrations/hibob/examples/mapping.yml

```diff
@@ -0,0 +1,21 @@
+resources:
+  - kind: list
+    selector:
+      query: .name | startswith("work.column_1659977338244") # Name of the List storing your teams' names (see lists.response.json)
+    port:
```

Comment:
seems like what you actually trying to achieve is way to declare to the kind what list to query from the api. 
we support ways to pass the api query params through the kind to the resync in order to perform more efficient queries. 

[see this pr](https://github.com/port-labs/ocean/pull/1315) for example on how we added support to pass query filters to integration that then are being applied to the api request.